supporting_content/metadata/metadata.md

```diff
@@ -0,0 +1,127 @@
+# Metadata Example API
+
+## Example Data Generator Metadata
+
+In this example the billing data generator's FOCUS metadata API is queried for the provider metadata.
+
+#### Endpoint: <api_root>/FOCUS/metadata/data_generator
+#### Example Request:
+
+    Method: GET 
+    Endpoint : <api_root>/FOCUS/metadata/data_generator
+####
+
+#### Response
+```
+{
+	"DataGenerator": "awesome_corp",
+    "DataGeneratorVersion": "1.0.1",
+	"ProviderTagPrefixes": ["awecorp", "ac"]
+}`
+```
+
+## Example Schema Metadata
+
+In this example, the billing data includes two different structures of data. An older schema for data previously provided, and a newer schema for more recent data. Both schemas metata is provided. 
+
+### API
+
+#### Endpoint: <api_root>/FOCUS/metadata/schemas 
```

Comment:
As the concept of Schema ID has been removed, in favor of keeping only the Data Generator Version, I wonder whether an API would rather list the Data Generator Versions and the associated schemas, than list the Schemas.

Reply:
The dropping of schemaId breaks the usefullnes of the schema metadata. How is data generator version going to help. It is a version of what? A version of AWS?

Reply:
Agreed to merge this PR and open a related issue for this discussion